edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss

```diff
@@ -1,7 +1,358 @@
+.edit-post-form input[type=text].field-input,
+.edit-post-form input[type=text].wrapper-custom-select {
+  color: $primary-font-color;
+}
+
+div.wmd-panel.wmd-preview p {
+  color: $inverse;
+}
+
+
+
+//static-tamplate
+
+section.container.about {
+  border-left: 1px solid $primary;
+  margin-top: 50px;
+  margin-bottom: 30px;
+
+  h1,
+  h2,
+  a, {
+    color: $primary;
+  }
+
+  ol {
+    list-style-type: none;
+  }
+}
+
+// Rocket chat tab service
+
+iframe.rocket-chat-view {
+  width: 100%;
+  height: 500px;
+  border: none;
+}
+
+.rocket_chat_error_message {
+  text-align: center;
+  color: red;
+}
+
+.alert.warning .action-secondary,
+.notification.warning .action-secondary,
+.prompt.warning .action-secondary {
+  color: $primary;
+}
+
+.course-info .pay-now-button.btn {
+  background-color: $pearson-dark-blue-courseware;
+  border: 1px solid $pearson-dark-blue-courseware;
+  padding: 10px 0;
+  width: 40%;
+  display: block;
+  font: normal 1.2rem/1.6rem Open Sans;
+  text-align: center;
+}
+
+.modal .inner-wrapper p {
+  font-size: 0.9em;
+}
+
+.modal .inner-wrapper .close-modal:focus {
+  outline: none;
+}
+
+.u-field-date_joined .u-field-title {
+  cursor: text;
+}
+
+.view-profile .tabpanel {
+  .badge-set-display {
+    min-width: auto;
+
+    .badge-list {
+      padding: 0;
+    }
+
+    .badge-display {
+      width: 100%;
+    }
+  }
+}
+
+@media (min-width: 768px) {
+  .view-profile .tabpanel {
+    .badge-set-display {
+      min-width: auto;
+    }
+  }
+}
+
+.signin-logo {
+  display: block;
+  width: 400px;
+  right: 110px;
+  position: absolute;
+  top: 100px;
+}
+
+.signin-container {
+  background: white;
+  padding: 30px;
+  border-radius: 10%;
+  box-shadow: 3px 2px 2px 5px #bcb9b9;
+  position: absolute;
+  right: 50px;
+  top: 250px
+}
+
+.signing-title {
+  text-align: center;
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+.login-register .form-wrapper .login-providers {
+  display: flex;
+  text-align: left;
+  background:  #f8f8f8;
+  padding: 10px 15px 40px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 2px 2px 2px #eee;
+  border-bottom: 0.1px solid #919191;
+}
+
+.login-register-content {
+  h2 {
+    @extend %t-title4;
+    font-size: 1.2em;
+    margin-top: $baseline;
+    letter-spacing: normal;
+    font-family: $font-family-sans-serif;
+    color: $uxpl-blue-hover-active;
+  }
+}
+
+.login-register .status {
+  margin: 0;
+}
+
+.email-email .label-text,
+.password-password .label-text {
+  display: block;
+  margin: 10px 0;
+  height: 0;
+}
+
+.form-field input.input-block {
+  height: 45px;
+  padding: 15px;
+  margin-bottom: 5px;
+}
+
+.login-register div.toggle-form {
+  margin-top: 15px;
+  text-align: center;
+}
+.login-register .action-primary {
+  width: 100%;
+  display: block;
+}
+
+.background-login {
+  background: url('/static/pearson-vue-theme/images/CertPREP-Sign-In-background.jpg');
+  height: 100vh;
+  width: 100%;
```

Comment:
Why not 100vw? @luismorenolopera

Reply:
@Squirrel18 It was required to see an image and a text in the lower part, so the size was modified vertically